Makefile

```diff
@@ -173,15 +166,15 @@ $(Worker_Enclave_u_Object): service/Enclave_u.o
 	$(AR) rcsD $@ $^
 	cp $(Worker_Enclave_u_Object) ./lib
 
-$(Worker_Name): $(Worker_Enclave_u_Object) $(Worker_SRC_Files)
+$(Worker_Name): $(Worker_Enclave_u_Object)
```

Comment:
I think this does not work, it will only rebuild the worker now when the `Worker_Enclave_u_Object` changed, but the changes in any other file are ignored, aren't they?

I think we either need to:
1. remove all build deps here in order to always build it
2. we simply find all files in the repository that have changed and exclude the `target` and the the `enclave-runtime/target` directory in the find process.

Reply:
Hmm yes you might be right, didn't realize that when looking at it.